.github/workflows/ci.yml

```diff
@@ -120,38 +120,32 @@ jobs:
         working-directory: integration-tests
         run: cargo test --release --locked --features "${{ matrix.features }}"
 
-  api-tests:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-
-      - uses: "./.github/actions/init"
-
-      - name: Run tests
-        working-directory: pop-api
-        run: cargo test --release --locked --all-features
-
-  api-integration-tests-devnet:
+  pop-api-integration-tests:
```

Comment:
Minor: not sure why needs to be prefixed with pop when its obvious by the repo/context. Crate name makes sense though, so feel free to ignore.

Reply:
https://github.com/r0gue-io/pop-node/pull/434/commits/b4ba54746511858e051486698eaa26dc07a106f1